js/memoryMonitor.js

```diff
@@ -33,26 +35,47 @@ function gatherMemory() {
  * application to free up some memory consumption
  * 
  * @param memoryInfo
+ * @param cpuUsage
  */
-function optimizeMemory(memoryInfo) {
+function optimizeMemory(memoryInfo, cpuUsage) {
     const memoryConsumed = (memoryInfo && memoryInfo.workingSetSize / 1024) || 0;
     const canReload = (!reloadedTimeStamp || (new Date().getTime() - reloadedTimeStamp) > reloadThreshold);
 
-    if (memoryConsumed > maxMemory && systemIdleTime.getIdleTime() > maxIdleTime && canReload) {
+    if (memoryConsumed > maxMemory
+        && systemIdleTime.getIdleTime() > maxIdleTime
```

Comment:
Fix linting please

Reply:
I don't see any linting issue here. Can you please let me know what is missing?